docs/source/gbd2017_models/concept_models/vivarium_swissre_breastcancer/concept_model.rst

```diff
@@ -277,219 +284,174 @@ A closed cohort of 100,000 male and female total simulants from age 15 to 95 wil
++------+------------+---------------------------+-----------------------------------------------------------------------------------------------------------------+                        |
+| LCIS | prevalence | | MarketScan (2017)       | | prevalence ratio of LCIS                                                                                      |                        |        
+|      |            | | Como (2017)             | | x                                                                                                             |                        |
+|      |            | | Forecasted (2020-2040)  | | prev_c429                                                                                                     |                        |
++------+------------+---------------------------+-----------------------------------------------------------------------------------------------------------------+                        |
+| LCIS | excess     | None                      | 0                                                                                                               |                        |  
+|      | mortality  |                           |                                                                                                                 |                        |             
++------+------------+---------------------------+-----------------------------------------------------------------------------------------------------------------+------------------------+
+| BC   | prevalence | | Como (2017)             | prev_c429                                                                                                       | see breast cancer      | 
+|      |            | | Forecasted (2020-2040)  |                                                                                                                 | model link below       |
++------+------------+---------------------------+-----------------------------------------------------------------------------------------------------------------+                        |
+| BC   | excess     | | codcorrect (2017)       | :math:`\frac{\text{deaths_c429}}{\text{pop}\times\text{prev_c429}}`                                             | Use 2017               |
+|      | mortality  | | Forecasted (2020-2040)  |                                                                                                                 | sources for 2017;      |                  
++------+------------+---------------------------+-----------------------------------------------------------------------------------------------------------------+ use forecast           |
+| BC   | disability | YLD appendix              | :math:`\displaystyle{\sum_{s\in\text{seq_c429}}}\scriptstyle{\text{disability_weight}_s\,\times\,\text{prev}_s}`| sources for            |   
```

Comment:
we can get a prevalence weighted average of disability weight for breast cancer if we dividing this measure by sum of prevalence_sequelae under breast cancer.